Makefile.am

```diff
@@ -395,6 +395,9 @@ run-rpm-tests-only: rpms
 	$(MAKE) TEST_SUITE_LOG=test-suite.log RPM_PATH=$(RPM_BUILD_DIR) \
 			ROOT_ANACONDA_PATH=$(abs_srcdir) TESTS=rpm_tests/rpm_tests.sh check
 
+cockpit-tests: rpms
```

Comment:
I think we will eventually want to be able to run these tests in a container, right? @jkonecny12?

Reply:
Yep, definitely!

Reply:
I have created an issue for that.